network-api/networkapi/wagtailpages/pagemodels/research_hub/landing_page.py

```diff
@@ -11,8 +20,35 @@ class ResearchLandingPage(foundation_metadata.FoundationMetadataPageMixin, wagta
         'ResearchAuthorsIndexPage',
     ]
 
+    intro = models.CharField(
+        blank=True,
+        max_length=250,
+    )
+
+    content_panels = [
+        FieldPanel(
+            'title',
+            classname='full title',
+            widget=TitleWidget(attrs={"class": "max-length-warning", "data-max-length": 60})
+        ),
```

Comment:
**Suggestion:** To avoid having to redefine the `title` field this elaborately, it is very common in Wagtail to do 
```python
content_panels = wagtail_models.Page.content_panels + [
    ...  # list the custom panels you want to use. 
]
```

Reply:
Updated!